src/Containers/CadView.jsx

```diff
@@ -188,6 +194,47 @@ export default function CadView({
   }, [location, model])
   /* eslint-enable */
 
+  // Fetch issues/notes
```

Comment:
Per discussion, let's move this hook to `Notes/NotesControl.jsx` for inclusion in OperationsGroup, like the other *Control components.  That way it's loaded once on application load.  The hook deps can be passed in as needed

Reply:
done and moved

Reply:
Removed from PR.